### PR TITLE
Bump SDK version and change walletapi to auth

### PR DESCRIPTION
**Pull request by nicoback2**
### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/services/audius-sdk/audiusSdk.ts [hunk 1]
```diff
@@ -34,6 +34,9 @@ const initSdk = async () => {
             }
           )
         },
+        signTransaction: async () => {
+          return 'Not implemented'
```
Comment (nicoback2) [inline packages/web/src/services/audius-sdk/audiusSdk.ts:38]: implementing in [this pr](https://github.com/AudiusProject/audius-client/pull/3396/files), just wanted to minimize the changes here

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
looks great! mind if i try doing the lock-file change? i think it should ideally be around +100 changes

**Review by rickyrombo — APPROVED** [review]

**Review by dylanjeffers — APPROVED** [review]
okay should be good now diff wise :)

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--bump-sdk

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--bump-sdk

**Pull request merged**